### PR TITLE
Improve response time when press menu icon...

### LINKED ISSUE #2362: Mobile: use "ng-click" in the sidebar and to close popup
To avoid delay in mobile devices

### PR DESCRIPTION
**Pull request by cmgustavo**
and Remove transition effects when closing modals.

fixes #2362

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,7 +129,7 @@ <h1 class="title ellipsis" ng-show="$root.iden && $root.wallet  && !$root.hideWa
           ng-class="{'main':$root.iden && !$root.starting && !$root.hideNavigation  && !$root.wpInputFocused}" 
           ng-view></section> 
 
-        <a class="exit-off-canvas"></a>
+        <a class="close-menu" ng-click="swipe()"></a>
```
Comment (matiu) [inline index.html]: we should decide whether wi will call swipe() functions or openMenu() functions on the HTML. It does not make sense to call swipe() and openMenu(). It should be `openMenu()` and `closeMenu()`.

Comment (cmgustavo) [inline index.html]: Updated. Thanks for your suggestion.

**Comment by cmgustavo** [comment]
Updated

**Comment by cmgustavo** [comment]
Updated

**Pull request merged**